tests/bootstrap.php

```diff
@@ -14,3 +14,12 @@
 EOT
     );
 }
+
```

Comment:
I wasn't sure if I could leverage `use` statements in this bootstrap file. If I get time, I will try to add them before the meetup.

Reply:
Yeah, you can leverage `use` statements here.

Reply:
Sweet! I will /use/ them soon :)